third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -109,6 +183,8 @@ const SEPARATE_BASELINE: &[&str] = &[
     "no-v1-comparison/assert_one.move",
     // Flaky redundant unused assignment error
     "no-v1-comparison/enum/enum_scoping.move",
+    "/lambda/",
```

Comment:
Could we add a comment for these new entries like in the cases above? Otherwise, it seems like "flaky redundant unused assignment error" applies to these two entries.

Reply:
Done.